src/components/Calculator.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import './Calculator.css';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calc">
+        <div className="calc__input-group">
+          <input className="calc__input" type="text" value="0" readOnly />
+        </div>
+        <ul className="rows">
+          <li className="row">
+            <button className="btn-default" type="button">
+              AC
+            </button>
+            <button className="btn-default" type="button">
+              +/-
+            </button>
+            <button className="btn-default" type="button">
+              %
+            </button>
+            <button className="btn-default btn-orange" type="button">
+              %
```

Comment:
- [ ] You duplicated the "%" sign, but you should have this instead "&div;". Kindly fix that.

  ![image](https://user-images.githubusercontent.com/36542986/211525546-84a743bf-43ba-4afe-b6b8-4ba34fe792a5.png)

Reply:
Hi @TedLivist 👋,   Thanks a lot for valuable suggestions. 🙏  I will appy your suggestions in my project.  🙂